app/views/articles/show.html.erb

```diff
@@ -224,6 +225,11 @@
   <% end %>
 <% end %>
 
+<div class="mod-actions-menu"></div>
+<div id="mod-actions-menu-btn-area"></div>
+<div class="flag-user-modal-container hidden"></div>
+<div class="modal-snackbar-container"></div>
+
```

Comment:
Feels like this should be behind some IF statement for mods only? Also, is this a good practice to have these kind of empty containers/placeholders anyway?

Also, would that conflict anyhow with snackbar we already have on that view? cc @nickytonline

Reply:
Tested the subscribe button, doesn't seem to affect the snackbar there 👍 

I agree that we should have some if statement here, but not sure how we would hide it behind that statement since we can't check for `current_user` because of caching. 😕 I think for now it's okay to have this here and then move it later once we refactor to using the modal + snackbar from Preact.